packages/commonwealth/client/scripts/views/pages/new_proposal/index.tsx

```diff
@@ -57,13 +72,13 @@ const NewProposalPage = (props: NewProposalPageProps) => {
   }
 
   // check if module is still initializing
-  const c = proposalSlugToClass().get(internalType) as ProposalModule<
+  const c = proposalSlugToClass()?.get(internalType) as ProposalModule<
     any,
     any,
     any
   >;
 
-  if (!c.ready) {
+  if (!c || !c.ready) {
     app.chain.loadModules([c]);
```

Comment:
This change makes it possible to send `undefined` into `loadModules`, to unblock the cow-loader render. But I didn't find any communities with modules to test. Any recommendations or concerns here?

Reply:
Those modules should only exist on substrate-based communities, so edgeware and kusama are two good test targets.

Reply:
For Cosmos, we can test on the juno or chihuahua communities. We can test on the tribe community for compound/open zeppelin, and aave or dydx for aave. NEAR we don't need to worry about right now.

Reply:
@jnaviask I tested juno, chihuaha, aave, and dydx. 

For Tribe, Edgeware, and Kusama, as far as I can tell, the behavior is the same as Prod. But I may be missing some context on how these should work for community members,

Reply:
Edgeware and Kusama (polkadot) have several different types of proposals, but we're deprecating support for creating them. So, no need to fix. Thanks for the info.